engine/apps/api/views/alert_group.py

```diff
@@ -237,10 +240,14 @@ def get_serializer_class(self):
 
     def get_queryset(self):
         # no select_related or prefetch_related is used at this point, it will be done on paginate_queryset.
+        alert_receive_channels_ids = list(
```

Comment:
_nit_; consider extracting the `alert_receive_channels_ids` logic into a method on `AlertGroupTeamFilteringMixin` (since it's reused twice)